prediction_market_agent/agents/microchain_agent/call_api.py

```diff
@@ -57,4 +53,27 @@ def __call__(
         return response.text
 
 
-API_FUNCTIONS: list[type[Function]] = [CallAPI]
+class SendTelegramMessage(Function):
+    @property
+    def description(self) -> str:
+        return "Use this function to send message on Telegram."
+
+    @property
+    def example_args(self) -> list[str]:
+        return ["123", "Hello from your telegram bot!"]
+
+    def __call__(
+        self,
+        chat_id: str,
+        message: str,
+    ) -> str:
+        url = f"https://api.telegram.org/bot{APIKeys().telegram_bot_key.get_secret_value()}/sendMessage?chat_id={chat_id}&text={message}"
+        response = requests.get(url)
+        response.raise_for_status()
+        return response.text
+
+
+API_FUNCTIONS: list[type[Function]] = [
+    CallAPI,
+    SendTelegramMessage,
```

Comment:
Feels odd for the trader agent to have this in its functions by default. I've made this new issue to address this https://github.com/gnosis/prediction-market-agent/issues/350. How do you feel about not having this always registered by the trader/just-born agents until we solve that?

Reply:
It's not needed atm so no feelings at all 😄  Commented out